include/tester/ssp/Softphone.hpp

```diff
@@ -38,9 +37,9 @@ class Softphone
         }
     }
 
-    void inComingCall(const pj::OnIncomingCallParam &iprm)
+    void onInComingCall(const pj::OnIncomingCallParam &iprm)
     {
-        SSPCall *in_call = new SSPCall(&_account, iprm.callId); //TODO
+        SSPCall *in_call = new SSPCall(&_account,boundOnCallState, iprm.callId); //TODO
```

Comment:
rather than use this outside the if block, you should only create a new instance of the call if the call is inactive. by the way, you're else block is hanging up a non-existing call. so remove that else block.